### ISSUE TITLE
Add-on shuts down after a while

### ISSUE THREAD

**Issue by rosiaantje** [robsonke/hassio-addons#3]
Hi the add-on keeps going down after a while. If activated the watchdog restarts it..
But during restart if gives unavailable to all sensors.

version: 1.0.11
Home assistant version, past couple.. 

Within supervisor log:
`21-11-10 21:53:26 WARNING (MainThread) [supervisor.misc.tasks] Watchdog found a problem with c0d67ef2_amcrest2mqtt!
21-11-10 21:53:26 INFO (SyncWorker_3) [supervisor.docker.interface] Cleaning addon_c0d67ef2_amcrest2mqtt application
21-11-10 21:53:28 INFO (SyncWorker_3) [supervisor.docker.addon] Starting Docker add-on c0d67ef2/amd64-addon-amcrest2mqtt with version 1.0.11 `

In the addon log I don’t see that much as it starts new every restart..

wifi signal on ad110 is good and stable.

Am I doing something wrong? 

Config works as addon starts and posts updates to mqtt.

**Comment by robsonke** [comment]
Oh that's something new, I haven't seen that before. Will try to keep an eye on my logs. Might be related to https://github.com/dchesterton/amcrest2mqtt/issues/44

**Comment by rosiaantje** [comment]
@robsonke any thing I could do/check/test/log?

**Comment by robsonke** [comment]
No, not really. Container is stable with me, so I'd say it's something in amcrest2mqtt that fails and triggers a restart. Is your addon log saying something more useful?  Otherwise let's wait for a new release.

**Comment by rosiaantje** [comment]
Problem is addon restarts, so log starts over every time. Or there should be a way “keep” the log.. that I’m not aware of.. I referenced a pull request that has already been made about the ping check. In the issue you mentioned. Let’s hope that solves the problem.. although my connection to the amcrest looks stable..

**Comment by robsonke** [comment]
Open a "docker log containerid -f" in a console?
